test/functional/feature_help.py

```
@@ -49,6 +51,22 @@ def run_test(self):
         assert b'version' in output
         self.log.info(f"Version text received: {output[0:60]} (...)")
 
+        missing_datadir = Path(self.options.tmpdir) / "missing_crashinfo_datadir"
+        assert not missing_datadir.exists()
+
+        self.log.info("Start dashd with -printcrashinfo and a nonexistent datadir")
+        self.nodes[0].start(extra_args=[f"-printcrashinfo=invalid", f"-datadir={missing_datadir}"])
```

Flake8 (7.3.0)

[error] f-string is missing placeholders

(F541)